README.md

```diff
@@ -1,2 +1,4 @@
 # testyMcTestFace
-dsgdsgfgs
+
+
+A proper desc
```

Comment:
/ticket do that thing as well -d desscription here

Reply:
Here's your ticket: [ do that thing as well](https://ticketgen19.atlassian.net/browse/IKKI-62)